app.py

```diff
@@ -0,0 +1,61 @@
+import os
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+
+app.config.from_object(os.environ['APP_SETTINGS'])
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+from models import User
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+
+
+@app.route("/add_new_user")
+def add_user():
+    first_name = request.args.get('first_name')
+    last_name = request.args.get('last_name')
+    email = request.args.get('email')
+    street_address = request.args.get('street_address')
+    state = request.args.get('state')
+    zip = request.args.get('zip')
+    try:
+        user = User(
+            first_name=first_name,
+            last_name=last_name,
+            email=email,
+            street_address=street_address,
+            state=state,
+            zip=zip
+        )
+        db.session.add(user)
+        db.session.commit()
+        return "User added. user id={}".format(user.id)
+    except Exception as e:
+	    return(str(e))
+
+
+@app.route("/getall")
+def get_all():
+    try:
+        users = User.query.all()
+        return jsonify([e.serialize() for e in users])
+    except Exception as e:
+	    return(str(e))
+
```

Comment:
Should this path be `/users` and `/user/:id` to be more like what we have done in the past?